collector_test.go

```diff
@@ -15,23 +15,28 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func getInstanaCollector() instana.TracerLogger {
+	c, _ := instana.GetCollector()
+	return c
+}
+
 func Test_Collector_Noop(t *testing.T) {
-	assert.NotNil(t, instana.C, "instana.C should never be nil and be initialized as noop")
+	assert.NotNil(t, getInstanaCollector(), "instana collector should never be nil and be initialized as noop")
 
-	sc, err := instana.C.Extract(nil, nil)
+	sc, err := getInstanaCollector().Extract(nil, nil)
 	assert.Nil(t, sc)
 	assert.Error(t, err)
-	assert.Nil(t, instana.C.StartSpan(""))
-	assert.Nil(t, instana.C.LegacySensor())
+	assert.Nil(t, getInstanaCollector().StartSpan(""))
+	assert.Nil(t, getInstanaCollector().LegacySensor())
 }
 
 func Test_Collector_LegacySensor(t *testing.T) {
 	recorder := instana.NewTestRecorder()
 	c := instana.InitCollector(&instana.Options{AgentClient: alwaysReadyClient{}, Recorder: recorder})
 	s := c.LegacySensor()
-	defer instana.ShutdownSensor()
+	defer instana.ShutdownCollector()
 
-	assert.NotNil(t, instana.C.LegacySensor())
+	assert.NotNil(t, getInstanaCollector().LegacySensor())
```

Comment:
```suggestion
	assert.NotNil(t, c.LegacySensor())
```
I think it is better to remove the helper function altogether and use the instance directly. What do you think?